src/lib.rs

```diff
@@ -90,41 +100,107 @@ pub struct IronOxide {
     pub(crate) rng: Mutex<ReseedingRng<ChaChaCore, EntropyRng>>,
 }
 
+/// Result of calling `initialize_check_rotation`
+pub enum InitAndRotationCheck {
+    /// Initialization succeeded, and no requests for private key rotations were present
+    NoRotationNeeded(IronOxide),
+    /// Initialization succeeded, but some keys should be rotated
+    RotationNeeded(IronOxide, PrivateKeyRotationCheckResult),
+}
+
+impl InitAndRotationCheck {
+    /// Caller asked to check rotation on initialize, but doesn't want to handle the result.
+    /// Consider using [initialize](fn.initialize.html) instead.
+    pub fn unwrap(self) -> IronOxide {
```

Comment:
I thought we'd decided on a different name for this, but can't think of it right now. `unwrap` in other apis will throw if it's not the "good" value so it seems like a bad name for this context.

Reply:
I _think_ I remember that conversation...
Potential names might be
`discard_result()`
`skip_check()`

Reply:
How about `discard_check`? (I'm not trying to troll)

Reply:
or how about `skip_result`!? `discard_check` actually seems pretty good.